packages/client/src/components/hub/sidebar.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {Layout, Menu} from 'antd';
+import {Link} from 'react-router-dom';
+import {appPrefix} from 'utils/env';
+import { withRouter, RouteComponentProps } from 'react-router';
+
+type Props = RouteComponentProps & {};
+class Sidebar extends React.Component<Props> {
+  render() {
+    const {history} = this.props;
+    const pathKeyList = ['home', 'hub', 'job', 'schedule', 'model-deployment'];
+    let key = '';
+    pathKeyList.forEach((val) => {
+      if (history.location.pathname.split('/').includes(val)) {
+        key = val;
+      }
+    });
+
+    return (
+      <Layout.Sider style={{paddingTop: 64}}>
+        <Menu
+          theme="dark"
+          selectedKeys={[key]}
+        >
+          <Menu.Item key="home">
+            <Link to={`${appPrefix}console/g/default/home`}>
```

Comment:
In production, the `appPrefix=console/`. So the link should be `${appPrefix}console/g/${groupName}/home`